package.json

```diff
@@ -14,9 +14,9 @@
     "asar": "^0.13.0",
     "bower": "^1.3.12",
     "electron": "^1.6.1",
-    "electron-builder": "^17.0.1",
+    "electron-builder": "^18.8.1",
     "electron-icon-maker": "^0.0.3",
-    "electron-publisher-s3": "^17.0.2",
+    "electron-publisher-s3": "^19.0.1",
```

Comment:
Also it's worth noting that they are recommending going to 19.x and electron-updater 2.2.0 on the release page:

https://github.com/electron-userland/electron-builder/releases

```
Please upgrade electron-updater to 2.2.0 if you already use 2.x version.

Bug Fixes

electron-updater: update sign verification error (e713bbe), closes #1641
```